x-pack/plugins/apm/server/lib/services/get_services/get_legacy_data_status.ts

```diff
@@ -31,7 +30,7 @@ export async function getLegacyDataStatus(setup: Setup) {
     }
   };
 
-  const resp = await client('search', params);
+  const resp = await client.search(params, { includeLegacyData: true });
```

Comment:
Does this mean we're moving our "own" parameters to the second argument instead of attaching them to the query object?

Reply:
yes, exactly. I never felt good about missing our custom options in with the ES params.